src/app/project-detail/project-detail.component.ts

```diff
@@ -50,6 +53,17 @@ export class ProjectDetailComponent implements OnInit {
       error => console.log(error)
     );
   }
+  setDocumentUrl(project) {
+    const regex = /http(s)?:\/\/(www.)?/;
+    project.recent_activities.forEach(activity => {
+      if (!activity.documentUrl) {
+        return ;
+      }
+      if (!regex.test(activity.documentUrl)) {
```

Comment:
Do we really need to check the document URLs and set them to point to `hostnameEPIC` if we update the data on the database ([see here](https://github.com/bcgov/esm-server/pull/387))?